peer-pod-controller/controllers/peerpodconfig_controller.go

```diff
@@ -326,9 +325,9 @@ func (r *PeerPodConfigReconciler) getNodesWithLabels(nodeLabels map[string]strin
 
 	if err := r.Client.List(context.TODO(), nodes, listOpts...); err != nil {
 		r.Log.Error(err, "Getting list of nodes having specified labels failed")
-		return err, &corev1.NodeList{}
+		return &corev1.NodeList{}, err
 	}
-	return nil, nodes
+	return nodes, nil
```

Comment:
Since the method signature is changed, subsequent changes need to be done where it's called.

Reply:
I'm wondering why no error is thrown during compilation
```
err, nodesList := r.getNodesWithLabels(map[string]string{"node-role.kubernetes.io/worker": ""})
```

Reply:
Missed it due to some local issue on my side, fixed it. PTAL!